src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => a.toString() > b.toString(),
+  ) {
+    sortArray(this, compareFunction);
+
+    return this;
+
+    function sortArray(array, comparator) {
+      for (let i = 0; i < array.length; i++) {
+        for (let j = i + 1; j < array.length; j++) {
+          if (comparator(array[i], array[j]) > 0) {
+            [array[i], array[j]] = [array[j], array[i]];
+          }
+        }
+      }
+    }
   };
```

Comment:
Modifying the prototype of built-in objects like Array is generally discouraged as it can lead to conflicts and unexpected behavior in other parts of the code. Consider using a different approach, such as creating a utility function that takes an array as an argument and returns a sorted array.